<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shapeControlEditorQWindow</class>
 <widget class="QWidget" name="shapeControlEditorQWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="colorTargetsQGroup">
     <property name="title">
      <string>Colorize:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QCheckBox" name="colorSelectionQCheck">
        <property name="text">
         <string>Selection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="colorShapesQCheck">
        <property name="text">
         <string>Shapes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="colorAffecToQGroup">
     <property name="title">
      <string>Affect To:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="colorViewportQCheck">
        <property name="text">
         <string>Viewport</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="colorOutlinerQCheck">
        <property name="text">
         <string>Outliner</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="colorGridQLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="colorQButton_22">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 99);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="colorQButton_13">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="colorQButton_1">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="colorQButton_24">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(161, 106, 48);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="colorQButton_3">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(153, 153, 153);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="colorQButton_14">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="colorQButton_5">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 4, 96);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="colorQButton_16">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="colorQButton_2">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(64, 64, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="colorQButton_6">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="colorQButton_27">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 161, 93);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="colorQButton_12">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(153, 38, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="colorQButton_8">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(38, 0, 67);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="colorQButton_4">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(155, 0, 40);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QPushButton" name="colorQButton_30">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(111, 48, 161);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QPushButton" name="colorQButton_19">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(67, 255, 163);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="colorQButton_29">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 103, 161);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QPushButton" name="colorQButton_31">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(161, 48, 106);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="colorQButton_7">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 70, 25);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="colorQButton_18">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(100, 220, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="colorQButton_26">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(104, 161, 48);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="colorQButton_0">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(120, 120, 120);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="colorQButton_17">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QPushButton" name="colorQButton_10">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(138, 72, 51);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="colorQButton_23">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 153, 84);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="colorQButton_25">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(158, 161, 48);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="colorQButton_15">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 65, 153);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="10">
      <widget class="QPushButton" name="colorQButton_21">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 172, 121);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QPushButton" name="colorQButton_20">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 176, 176);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="colorQButton_28">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 161, 161);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="colorQButton_9">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 0, 200);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="colorQButton_11">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(63, 35, 31);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="colorUtilsQLayout">
     <item>
      <widget class="QPushButton" name="colorDisableQButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Disable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorRGBQButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>RGB</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
